build/pyinstaller/terminal.spec

```diff
@@ -19,6 +23,14 @@ build_type = (
 # Local python environment packages folder
 pathex = os.path.join(os.path.dirname(os.__file__), "site-packages")
 
+# Removing unused ARM64 binary
+binary_to_remove = pathlib.Path(
+    os.path.join(pathex, "_scs_direct.cpython-39-darwin.so")
+)
+print("Removing ARM64 Binary: _scs_direct.cpython-39-darwin.so")
```

Comment:
Is the print intentional here?

Reply:
@DidierRLopes Yes. While not absolutely necessary, it provides extra clarification during the build procedure

<img width="705" alt="Screen Shot 2022-10-03 at 7 09 46 PM" src="https://user-images.githubusercontent.com/53658028/193701888-8eed12bd-062f-45ab-b922-6fafe26184d9.png">